api/extras/README.md

```diff
@@ -0,0 +1,11 @@
+# Influx CLI - extra files supporting HTTP client
```

Comment:
We have `api/contract/overrides/` already, do we also need to add `api/extras/`? Could we merge the two and add this note about our changes to enum generation in `api/templates/README.md`?

Reply:
I don't think so - we don't want to update the main swagger in such a way that it causes the default generators to actually validate the enum (to avoid breaking cloud/other tooling).

The compromise is to have `api/cli-extras.gen.yml`, which has this (only) to convince the openapi generator that these types are schemas that need full enum generation, rather than in-line enums that only get minimal codegen:

```
components:
  schemas:
    ResourceEnumOSS:
      $ref: "./openapi/src/common/schemas/Resource.yml#/properties/type"
    ResourceEnumCloud:
      $ref: "./openapi/src/cloud/schemas/Resource.yml#/properties/type"
```

Importantly, if there are any references to Resource.yml higher in the file, these would not be fully expanded in `api/cli-extras.gen.yml` and so the enum code is not generated because then they count as inline. It is quite annoying.